about.html

```diff
@@ -0,0 +1,194 @@
+          alt="Background image"
+          class="bg-header-image"
+        />
+        <img
+          src="./utils/images/—Pngtree—line-decorative-design-pattern_3634817.png"
+          alt="Background image"
+          class="bg-header-image-rotate"
+        />
+        <div class="main-content">
+          <div class="main-content-headers flex-column-center">
+            <h2 class="special-color text-center">"Who are we?"</h2>
+            <h1 class="special-color text-center">
+              Asian Youth Program (AYP) CLUB ACTIVITIES
+            </h1>
+          </div>
+          <div class="header-content text-center margin-top-big">
+            <p>
+              Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center), offers mentoring to children impacted by domestic violence.
+            </p>
+          </div>
+          <div class="infos">
+            <p class="location light-black text-center">
+              <i
+                >Please Contact Us with Email for any further questions about
+                this activity!</i
+              ><br /><b>AYP@gmail.com</b>
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="about-us flex-column-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <img
+        src="https://securityconference.org/assets/media/e/5/csm_themen_cyber-security-and-technology_header_compr_2037a5dc78.jpg"
+        alt="about us community photo"
+        class="family-photo"
+      />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">
+          Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center), offers mentoring to children impacted by domestic violence.
+        </p>
+        <h4>
+          <br />
+          What do we promise to youths and adults?
+        </h4>
+        <p>
+          <br />
+          Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center).
+        </p>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <section class="past-activities" id="past-activities">
+      <h2 class="past-heading light-black heading-medium">
+        See the past Club Activities
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <p class="past-content light-black text-center">
+        Take a look at the last two years activities which took place in
+        Asia
+      </p>
+      <div class="activities">
+        <div class="past-first-activity act">
+          <div class="first-activity-content flex-column-center">
+            <h3>2021</h3>
+            <p>AYP Summit in Asia</p>
+          </div>
+        </div>
+        <div class="past-second-activity act">
```

Comment:
- [ ] the background image in the first card is missing, kindly add a background image 👍🏻 
<img width="755" alt="Screenshot 2022-06-29 at 17 17 09" src="https://user-images.githubusercontent.com/77038610/176445663-65fadedf-9255-4399-b065-88c68ec48585.png">